### PR TITLE
Revise resource specifications and recommendations

### PR DESCRIPTION
**Pull request by ag-adampike**
Our engineering team responsible for developing our SCIM bridge container image has spent the last several months investigating and refining how SCIM bridge performs at scale. We now have updated guidelines for recommended resources that should be allocated to the SCIM bridge container when provisioning large numbers of users and groups to 1Password.

This PR revises our deployment examples to:

- ensure that the default resource allocations for the SCIM bridge and Redis containers on each respective platform align with our base recommendations
- include guidelines for vertically scaling at higher volumes of provisioning activity
- remove any references to scaling the Redis container and recommend always using the default resource specification

### CODE REVIEW

**Review by black-bryan — APPROVED** [review]
🎉 LGTM

File: kubernetes/README.md [hunk 1]
```diff
@@ -427,3 +391,13 @@ kubectl set env deploy/op-scim-bridge OP_PING_SERVER=1
 ```
 
 No other endpoints (such as `/scim`) are exposed through this port.
+
+## Troubleshooting
+
+When updating cluster resources in place, Kubernetes will create a new revision alongside an existing workload before replacing the existing revision. If there are not enough available resources in the cluster on which to schedule both revisions, the update can hang until they become available. This can happening when vertically scaling a Deployment or updating to a new SCIM bridge image version.
+
+If there are enough resources in the cluster available to schedule the new Pod revision, you can restart the Deployment to evict the existing Pod, free up the resources, and allow the new Pod to start:
```
Comment (Penderworth) [inline kubernetes/README.md:399]:
```suggestion
If there are enough resources in the cluster available to schedule the new Pod revision, you can restart the deployment to evict the existing Pod, free up the resources, and allow the new Pod to start:
```

**Review by Penderworth — CHANGES_REQUESTED** [review]
Just two small updates and this is ready to go from the wording side of things.

File: kubernetes/README.md (hunk 1 above)
Comment (ag-adampike) [reply kubernetes/README.md:399]: This one is intended. Kubernetes resource types are proper nouns: Deployment, Pod, StatefulSet, etc. I am referring to the Deployment object here specifically.

Comment (Penderworth) [reply kubernetes/README.md:399]: Aha, TIL!

**Review by Penderworth — APPROVED** [review]

**Pull request merged**